docs/developing.md

```diff
@@ -77,19 +77,56 @@ The Actor tests use resmoke to set up a real MongoDB cluster and execute
 the test binary. The resmoke yaml config files that define the different
 cluster configurations are defined in `src/resmokeconfig`.
 
-```sh
-# Example:
-./run-genny resmoke-test --suites src/resmokeconfig/genny_standalone.yml
-```
-
 Each yaml configuration file will only run tests that are associated
-with their specific tags. (Eg. `genny_standalone.yml` will only run
-tests that have been tagged with the `[standalone]` tag.)
+with their specific tags. (Eg. `genny_sharded.yml` will only run
+tests that have been tagged with the `[sharded]` tag.)
 
 When creating a new Actor, `./run-genny create-new-actor` will generate a new test case
 template to ensure the new Actor can run against different MongoDB topologies,
 please update the template as needed so it uses the newly-created Actor.
 
+Currently, the code to implement resmoke-test assumes that it will be run by
+Evergreen. Therefore, to run resmoke-test locally, some modifications must be made.
+The commands to make these modifications on MacOS and on Linux are detailed below.
+
+These commands assume that `git status` last output line is
+`nothing to commit, working tree clean`, and that the current working directory is
+`genny/` i.e. the root of the repo.
+
+#### MacOS:
+1. `mkdir -p data/db && sed -i '' '/mongos,/s/mongos,/mongos, "--dbpathPrefix", "data\/db"/' src/lamplib/src/genny/tasks/run_tests.py`
```

Comment:
1. there is an extra pair of single quotes (`''`) should be deleted?
2. I'd suggest, similar to the next line, using `sed -i '/xxx/s!from!to!' so no need of escaping slash(es) in path
3. the "to" does not contain an trailing ",". I'd suggest adding that so that it won't fail even if more options are appended in the future.

In summary I think this command should be like this
`mkdir -p data/db && sed -i '' '/mongos,/s!mongos,!mongos, "--dbpathPrefix", "data/db", !' src/lamplib/src/genny/tasks/run_tests.py`

Reply:
MacOS's BSD implementation of `sed` is different from GNU/Linux's, hence the necessity for the extra pair of single quotes on MacOS. Took me a while to figure out why the Linux version of the command wasn't working on my Mac: https://stackoverflow.com/a/4247319

I'm a complete sed noob, so thank you for the suggested command! I've added it with the trailing space after the comma removed.